src/utilities/routes.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Convenience functions for centralized routing
+ * Separate into categories so we can loop over them for navigation/local footer as needed
+ */
+
+// Initiatives
+export const initiativesRoot = 'initiatives';
+
+export const initiatives = {
+  changingHumanExperience: {
+    name: 'Changing Human Experience',
+    path: `${initiativesRoot}/changing-human-experience`,
+  },
+  ethicsSocietyTech: {
+    name: 'Ethics, Society, and Technology',
+    path: `${initiativesRoot}/ethics-society-and-technology`,
```

Comment:
Have both the Themes and Initiatives in this format so we can loop over them if we want to